code/pages.py

```diff
@@ -0,0 +1,22 @@
+from talon import Module, actions
+
+# --- Tag definition ---
+mod = Module()
+mod.tag("pages", desc="Anything with page navigation")
+
+
+# --- Define actions ---
+@mod.action_class
+class Actions:
+    def page_current() -> int:
+        """Return current page number"""
+    def page_next():
+        """Go to next page"""
+        actions.user.page_jump(actions.user.page_current() + 1)
+    def page_previous():
+        """Go to previous page"""
+        actions.user.page_jump(actions.user.page_current() - 1)
```

Comment:
Do we want to do the `min` thing here that you did for chapters?

Reply:
I actually removed that because i didn't want the flickering of jumping from page 1 to page 1 (not really applicable in any of these apps anyways). I just forgot it in chapters.

Reply:
Ok I'm not sure I follow but do what you think makes sense here; was just flagging the discrepancy